charmcraft.yaml

```
@@ -12,6 +12,8 @@ parts:
   charm:
     build-packages:
       - git
+      - rustc
+      - cargo
```

At this point I think I'd rather get the binary. Otherwise it's 20-30 min packing.
Wdyt?

```suggestion
    charm-binary-python-packages:
      - jsonschema
```

---

Until @sergiusens patch that allows caching pip build artifacts land in charmcraft (maybe it already has?), that seems like a reasonable trade-off to not make it too tedious to iterate on the charm. We could set up our own fork to build the binary with more control.

---

Charmcraft does already cache built wheels across runs (with `charmcraft clean` eliminating that cache), but it will of course slow down CI and first build after a clean.